apps/core/models/core.py

```diff
@@ -1,7 +1,17 @@
+import math
+
+from django.core.validators import ValidationError
 from django.db import models
 
 
+def validate_aspect_ratio(image):
+    ratio = 1
+    if not math.isclose(image.width / image.height, ratio, rel_tol=1e-6):
+        raise ValidationError("The aspect ratio is not correct. The aspect ratio should be: " + str(ratio))
+
+
 class Testimonial(models.Model):
+    image = models.ImageField(upload_to="testimonials", blank=True, validators=[validate_aspect_ratio])
```

Comment:
Make sure that it looks decent without an image as well, otherwise add a default image.

Reply:
I added the default image functionality, but one still needs to be uploaded to the media folder which I don't have access to.